pkgs/top-level/all-packages.nix

```diff
@@ -5001,7 +5001,8 @@ let
     installLocales = config.glibc.locales or false;
   };
 
-  glibcLocales = callPackage ../development/libraries/glibc/2.19/locales.nix { };
+  # Not supported on Darwin
+  glibcLocales = if (! stdenv.isDarwin) then (callPackage ../development/libraries/glibc/2.19/locales.nix { }) else null;
```

Comment:
@edolstra I was under the impression that glibc locales were supported everywhere but on Darwin - wrong?